packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -24,16 +24,16 @@ import {
   SUCCESS_CODE,
   TOOLS_DIR
 } from '../constants';
+import { SObjectDescribe } from '../describe';
+import { nls } from '../messages';
 import {
   ChildRelationship,
   Field,
-  MAX_BATCH_REQUEST_SIZE,
   SObject,
   SObjectCategory,
-  SObjectDescribe
-} from '../describe';
-import { ConfigUtil } from '../describe/configUtil';
-import { nls } from '../messages';
+  SObjectRefreshSource
+} from '../types';
+import { ConfigUtil } from './configUtil';
```

Comment:
Was only being used by this module so move it

Reply:
@lcampos How about using **const connection = await workspaceContext.getConnection()** to get all connections?

Reply:
That is an option but I'm leaning more towards removing the code that gets the connection object (in following PRs) and have that be something the vscode extension provides to the `salesforcedx-sobject-faux-generator` module. This would be similar to what we already do on SDR and the Apex library.